### ISSUE TITLE
Tracker domain not set in new Tag Manager variable

### ISSUE THREAD

**Issue by mhh515** [digitalist-se/TrackerDomain#10]
The plugin works great for setting a custom URL in the standard and Tag Manager tracking scripts. However, when creating a new Tag Manager variable of type "Matomo Configuration", then the default value for the parameter "Matomo URL" in the section "Configure this variable" is set to the default URL, not to the one from the Tracker Domain setting.

Expected behaviour: when creating a new variable as explained above, the "Matomo URL" parameter should also default to the URL defined in the Tracker Domain settings.

**Comment by mikkeschiren** [comment]
Agree, this should be expcted behaviour, the behaviour is documented in the README (https://github.com/digitalist-se/TrackerDomain/blob/main/README.md#tag-manager-configuration-variable), but should be solved.
